bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4254,6 +4254,40 @@
             INSERT INTO role_role(child_role, parent_role) VALUES ('Appointments:FullAccess', 'Appointments:ManageAppointments');
         </sql>
     </changeSet>
+    <changeSet id="201802211437" author="ningosi">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM global_property where property = 'patientIdentifierSearch.matchMode';
```

Comment:
if we're going to have a precondition here I'd have it also check that property_value != 'ANYWHERE'